hack/run-ut.sh

```diff
@@ -2,6 +2,51 @@
 
 ROOT_DIR=$(git rev-parse --show-toplevel)
 
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:$ROOT_DIR/api/v2-c:$ROOT_DIR/bpf/deserialization_to_bpf_map
-export PKG_CONFIG_PATH=$ROOT_DIR/mk
-go test -v -vet=off -coverprofile=coverage.out ./pkg/...
\ No newline at end of file
+. $ROOT_DIR/hack/utils.sh
+
+function docker_run_go_ut() {
+    local container_id=$1
+    docker exec $container_id go test -v -vet=off -coverprofile=coverage.out ./pkg/...
+}
+
+function run_go_ut_local() {
+    bash $ROOT_DIR/build.sh
+    export PKG_CONFIG_PATH=$ROOT_DIR/mk
+    go test -v -vet=off -coverprofile=coverage.out ./pkg/...    
+}
+
+function run_go_ut_in_docker() {
+    prepare
+    container_id=$(run_docker_container)
+    build_kmesh $container_id
+    docker_run_go_ut $container_id
+    clean_container $container_id
+}
+
+function clean() {
+    make clean $ROOT_DIR
+    rm -rf $ROOT_DIR/coverage.out
+}
+
+# Running go ut with docker by default
+if [ -z "$1" -o "$1" == "-d"  -o  "$1" == "--docker" ]; then
+    run_go_ut_in_docker
+    exit
+fi
+
+if [ "$1" == "-l"  -o  "$1" == "--local" ]; then
+    run_go_ut_local
```

Comment:
I met an error running with `./hack/run-ut.sh -l`

Reply:
What is the error?